.lintstagedrc.json

```diff
@@ -3,5 +3,6 @@
     "prettier --write",
     "cspell --no-must-find-files"
   ],
-  "*.ts": ["eslint --fix", "cspell --no-must-find-files"]
+  "*.ts": ["eslint --fix", "cspell --no-must-find-files"],
+  "*.prisma": ["npx --no-install prisma format"]
```

Comment:
I think It's good to have spell check  `.prisma` schema files too

Reply:
Good idea.